src/main/java/uk/ac/ebi/intact/app/internal/model/core/elements/edges/SummaryEdge.java

```diff
@@ -28,6 +29,16 @@ public Map<Node, List<Feature>> getFeatures() {
         return features;
     }
 
+    public Boolean isNegative() {
```

Comment:
So, if any of the evidences in a summary edge is negative, the summary edge is negative? Is this what we do in Intact website?

Also, we can use streams, like
```
return getSummarizedEdges().stream().anyMatch(edge -> edge.isNegativeInteraction);
```